.github/pull_request_template.md

```diff
@@ -1,39 +1,52 @@
-## Description
+### Description
 
-A detailed explanation of the changes in your PR.
+<!-- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
+change me!
 
-## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
+### User-facing documentation
 
-If any of these don't apply, please comment below.
+(*must be* 2 items and both *must be* checked)
+<!-- Remove conflicting items that won't be checked. -->
 
-## Testing Performed
+- [ ] CHANGELOG is updated
+- [ ] CHANGELOG update is not needed
+- [ ] Documentation PR is created and linked above
+- [ ] Documentation is not needed
```

Comment:
```suggestion
- [ ] `CHANGELOG.md` is updated
- [ ] `CHANGELOG.md` update is not needed
- [ ] documentation PR is created and linked above
- [ ] documentation is not needed
```

Reply:
Maybe even a link to a [`CHANGELOG.md`](https://github.com/stackrox/stackrox/blob/master/CHANGELOG.md)